vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -5500,6 +5482,7 @@ unit:CentiM_H2O_4DEG_C
   qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-2D0 ;
   qudt:hasQuantityKind quantitykind:Unknown ;
   qudt:iec61360Code "0112/2///62720#UAB236" ;
+  qudt:symbol "cmH₂0 (4 °C)" ;
   qudt:uneceCommonCode "N14" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "centimetre of water (4 ºC)" ;
```

Comment:
Thanks for the cleanups. While you are at it, does this look weird to you?

Reply:
In what way? Spacing?